packages/wordpress-block-editor-config/src/settings/__experimental-fetch-link-suggestions.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * WordPress dependencies
+ */
+// @ts-ignore TODO
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+import { decodeEntities } from '@wordpress/html-entities';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Filters the search by type
+ *
+ * @typedef { 'post' | 'term' | 'post-format' } WPLinkSearchType
+ */
+
+/**
+ * @typedef WPLinkSearchOptions
+ *
+ * @property {?boolean}              isInitialSuggestions Displays initial search suggestions, when true.
```

Comment:
Is the intention to make these optional or to make the nullable?

Reply:
I was mostly trying to see if we could still allow an object default argument, while destructuring, since I wasn't intending on refactoring that much. 

I'll audit usage and try to tidy this.
```
const fetchLinkSuggestions = async (
	search,
	{ isInitialSuggestions, type, subtype, page, perPage: perPageArg } = {},
	{ disablePostFormats = false } = {}
) => {
```

I'll try and tidy that

Reply:
This one should be resolved by https://github.com/WordPress/gutenberg/commit/9cfa3453cffa7ca35e1e3fbb00e96fd62ec46662